designs/waralee/src/backpocket.mjs

```diff
@@ -1,54 +1,83 @@
 import { pantsProto } from './pantsproto.mjs'
 
-function waraleeBackPocket({
-  options,
-  measurements,
-  Point,
-  Path,
-  points,
-  paths,
-  Snippet,
-  snippets,
-  complete,
-  paperless,
-  macro,
-  sa,
-  part,
-}) {
-  if (false == options.backPocket) {
-    return part
-  }
+export const backPocket = {
+  name: 'waralee.backPocket',
+  after: pantsProto,
+  draft: ({
+    options,
+    measurements,
+    Point,
+    Path,
+    points,
+    paths,
+    Snippet,
+    snippets,
+    macro,
+    sa,
+    store,
+    expand,
+    units,
+    part,
+  }) => {
+    if (false == options.backPocket) {
+      return part.hide()
+    }
+    const pocketDepth = options.backPocketDepth * measurements.crotchDepth
+
+    if (!expand) {
+      // Expand is on, do not draw the part but flag this to the user
```

Comment:
Small nitpick, but this comment says *expand is **on**, do...* when it should say **off**.

I know, I know. That was my original typo, but I've fixed it since ;)

https://github.com/freesewing/freesewing/blob/006a1f0ac4f18d9ee6383d8aa322b52b0db3ecfb/designs/aaron/src/arm-binding.mjs#L27